src/test/java/com/revature/personalfinance/service/AuthenticatorTests.java

```diff
@@ -0,0 +1,80 @@
+package com.revature.personalfinance.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class AuthenticatorTests {
+
+	// We will need more accurate JWTs that we can manipulate before being able to complete the testing
+	String validJWT = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9."
```

Comment:
I'm concerned about this testing method. Using a hard coded JWT that will always be valid seems like it would cause a security issue because anyone could then grab that jwt from the repo and use it to access the application. You may want to reach out to @KennethDavis391 for his input.

Reply:
That token does not contain valid user credentials, it is a mock token only used for testing purposes. No tokens containing valid user credentials will ever be stored, nor will they be hard-coded anywhere in the application.